autocam.sh

```diff
@@ -0,0 +1 @@
+export CAMERA=ZWO
```

Comment:
p.s. this file is auto created each time the allsky service is (re)started and used only temporarily for the current "service session". seems like scripts read the config, and autoselection need to be saved somewhere